### ISSUE TITLE
can I control the air conditioner through ble?

### ISSUE THREAD

**Issue by zhy2020** [yoziru/tesla-ble#13]
Hello, can I control the air conditioner through ble?

**Comment by mrd0n** [comment]
Yes - The code for adding the HVAC and Steering Wheel heater switches (and Sentry Mode) have ben contributed and ready to be merged.   In the meantime, I have a fork with these merged and working:   https://github.com/mrd0n/esphome-tesla-ble.git and https://github.com/mrd0n/tesla-ble.git  

These contributions are from [@Aaneisch ](https://github.com/aneisch) and [@herrfrei ](https://github.com/herrfrei)

**Comment by yoziru** [comment]
Done in https://github.com/yoziru/tesla-ble/pull/26, all actions now exposed
